specification/sql/resource-manager/Microsoft.Sql/preview/2018-06-01-preview/instancePools.json

```diff
@@ -0,0 +1,534 @@
+            "required": true,
+            "type": "string"
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully retrieved the specified instance pool.",
+            "schema": {
+              "$ref": "#/definitions/InstancePool"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 404 InstancePoolNotFound - An instance pool cannot be found."
+          }
+        },
+        "x-ms-examples": {
+          "Get an instance pool": {
+            "$ref": "./examples/GetInstancePool.json"
+          }
+        }
+      },
+      "put": {
+        "tags": [
+          "InstancePools"
+        ],
+        "description": "Creates or updates an instance pool.",
+        "operationId": "InstancePools_CreateOrUpdate",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ResourceGroupParameter"
+          },
+          {
+            "name": "instancePoolName",
+            "in": "path",
+            "description": "The name of the instance pool to be created or updated.",
+            "required": true,
+            "type": "string"
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "description": "The requested instance pool resource state.",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/InstancePool"
+            }
+          },
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Successfully updated the instance pool.",
+            "schema": {
+              "$ref": "#/definitions/InstancePool"
+            }
+          },
+          "default": {
+            "description": "*** Error Responses: ***\n\n * 400 InvalidResourceRequestBody - The resource or resource properties in the request body is empty or invalid.\n\n * 400 MissingSkuName - Sku name is rerquired.\n\n * 400 InstancePoolRequestedVcoreCountIsInvalid - Requested VCore count for instance pool is invalid.\n\n * 400 InstancePoolRequestMissingSku - Instance Pool request is missing Sku.\n\n * 400 InstancePoolRequestMissingSkuTier - Instance Pool request is missing Sku Tier.\n\n * 400 InstancePoolRequestMissingSkuFamily - Instance Pool request is missing Sku Family.\n\n * 400 InstancePoolRequestedSubnetResourceIdIsInvalid - Subnet Resource Id for instance pool is Null or empty.\n\n * 400 InstancePoolRequestedLicenseTypeIsInvalid - License Type for instacne pool is Null or empty.\n\n * 409 InstancePoolBusy - An instance pool is busy with another ongoing operation."
```

Comment:
rerquired -> required

Reply:
acked, found out that they are in the exception response in general. I'll fix it.

Reply:
Fixed typo